input/fsh/DEF_Rulesets.fsh

```diff
@@ -71,16 +72,45 @@ RuleSet: RadiotherapyProcedureCommon
 * performedPeriod.start ^short = "The date and time when the first therapeutic radiation was delivered."
 * performedPeriod.end MS
 * performedPeriod.end ^short = "An end date is expected if the status is 'stopped' or 'completed'"
+* insert OpenProfileBasedSlicing(usedReference)
+* usedReference contains
+    treatmentDevice 0..* MS
+* usedReference[treatmentDevice] only Reference(RadiotherapyTreatmentDevice)
+* usedReference[treatmentDevice] ^short = "RadiotherapyTreatment Devices used as part of therapy."
+* usedReference MS
+* insert OpenProfileBasedSlicingSubfield(focalDevice,manipulated)
+* focalDevice contains
+    seedDevice 0..* MS
+* focalDevice[seedDevice].manipulated only Reference(RadiotherapySeedDevice)
+* focalDevice[seedDevice] ^short = "Radiotherapy Seed Devices used as part of therapy."
 
 RuleSet: RadiotherapyPrescriptionsCommon
 * insert RadiotherapyRequestCommon
 * intent = ReqIntent#original-order "Original Order"
 * replaces MS
 * replaces ^short = "Previous retired prescription that is replaced by this prescription"
+* insert OpenProfileBasedSlicing(performer)
+* performer contains
+    treatmentDevice 0..* MS and
+    seedDevice 0..* MS
+* performer[treatmentDevice] only Reference(RadiotherapyTreatmentDevice)
+* performer[seedDevice] only Reference(RadiotherapySeedDevice)
+* performer MS
+* performer[treatmentDevice] ^short = "RadiotherapyTreatment Devices used as part of therapy."
+* performer[seedDevice] ^short = "Seed Devices used as part of therapy."
 
```

Comment:
We should use singular, and space missing in "Radiotherapy Treatment Device used ...", "Radiotherapy Seed Device used ..."